metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java

```diff
@@ -39,6 +39,8 @@
     private static final String CSV_MAP = "src/test/resources/org/metafacture/metafix/maps/test.csv";
     private static final String TSV_MAP = "src/test/resources/org/metafacture/metafix/maps/test.tsv";
 
+    private static final String LOOKUP = "lookup('title.*',";
```

Comment:
Minor: for tests, I often like to keep redundancies if they add clarity, to get kind of contained test cases.

Reply:
Well, what can I say? Got overpowered by my OCD ;)